client/packages/electron/src/electron.ts

```diff
@@ -8,14 +8,45 @@ import {
 } from '@openmsupply-client/common/src/hooks/useElectronClient';
 import storage from 'electron-data-storage';
 
+const QUERY_INTERVAL = 1000;
 const SERVICE_NAME = '_omsupply._tcp.local';
 const SERVICE_TYPE = 'TXT';
-const QUERY_INTERVAL = 1000;
+
+class DiscoveryRunner {
+  isDiscovering: boolean;
+  intervalHandle?: NodeJS.Timer;
+
+  constructor() {
+    this.isDiscovering = false;
+    this.intervalHandle = this.intervalHandle = setInterval(() => {
```

Comment:
double intervalHandle assignment...

why not start the setInterval in the start method()?

Reply:
🤣  oops!

and 🤦 who wrote this code??! what was I thinking? Changed in 28a287d30
thanks